odoo/addons/base/models/ir_cron.py

```diff
@@ -189,8 +194,8 @@ def _get_all_ready_jobs(cls, cr):
         return cr.dictfetchall()
 
     @classmethod
-    def _acquire_one_job(cls, cr, job_ids):
-        """ Acquire one job for update from the job_ids tuple. """
+    def _acquire_job(cls, cr, job_id):
```

Comment:
@odony this patch fixes a nasty serialization error in the cron worker(s).
The only bit that worries me is this API change.  That being said, it looks reasonable to me.
What's your opinion about it?